kong/plugins/oauth2/daos.lua

```diff
@@ -51,7 +50,6 @@ local OAUTH2_TOKENS_SCHEMA = {
     access_token = { type = "string", required = false, unique = true, queryable = true, immutable = true, func = generate_if_missing },
     refresh_token = { type = "string", required = false, unique = true, queryable = true, immutable = true, func = generate_refresh_token },
     expires_in = { type = "number", required = true },
-    authenticated_username = { type = "string", required = false },
     authenticated_userid = { type = "string", required = false },
```

Comment:
This should be marked as queryable if you're leaving the INDEX on it so it avoids the `ALLOW FILTERING` in queries